<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1Q9C5PA" ignored="0" name="BankingApp Test(failed)" tests="4" failures="3" timestamp="09 Aug 2021 11:13:30 GMT" time="43.195" errors="0">
  <testcase name="loginDDT" time="9.169" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="8.062" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.94" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginTest" time="2.129" classname="com.inetBanking.testCases.TC_LoginPage_001"/>
</testsuite> <!-- BankingApp Test(failed) -->
